cmd/lakefs/cmd/run.go

```diff
@@ -50,301 +34,102 @@ type Shutter interface {
 	Shutdown(context.Context) error
 }
 
-func newLDAPAuthenticator(cfg *config.LDAP, service auth.Service) *auth.LDAPAuthenticator {
-	const (
-		connectionTimeout = 15 * time.Second
-		requestTimeout    = 7 * time.Second
-	)
-	group := cfg.DefaultUserGroup
-	if group == "" {
-		group = auth.ViewersGroup
-	}
-	return &auth.LDAPAuthenticator{
-		AuthService:       service,
-		BindDN:            cfg.BindDN,
-		BindPassword:      cfg.BindPassword,
-		DefaultUserGroup:  group,
-		UsernameAttribute: cfg.UsernameAttribute,
-		MakeLDAPConn: func(_ context.Context) (*ldap.Conn, error) {
-			c, err := ldap.DialURL(
-				cfg.ServerEndpoint,
-				ldap.DialWithDialer(&net.Dialer{Timeout: connectionTimeout}),
-			)
+func NewRootCmd() *cobra.Command {
```

Comment:
It's not the `RootCmd`. Can you revert this function change?

Reply:
Ops, yes. If you have nothing in comment I will leave the function pattern. It's easy to to test, I will just rename it as NewRunCmd()